hedera-node/src/main/java/com/hedera/services/context/ServicesContext.java

```diff
@@ -879,6 +891,19 @@ public HcsAnswers hcsAnswers() {
 		return hcsAnswers;
 	}
 
+	public EntityStore entityStore() {
```

Comment:
Let's add a comment here about the lifecycle of the store and anything else of value you can think of that isn't already documented elsewhere. Maybe just a sentence or two.